### ISSUE TITLE
Consider changing "Manage credentials" permission dialog language to refer to wallet selector instead

### ISSUE THREAD

**Issue by dlongley** [credential-handler/authn.io#150]
We should come up with better language than "Manage credentials". A wallet / credential handler provider is simply asking to appear in a wallet selector for / when credential interactions occur in the future (on other websites).

**Comment by djscruggs** [comment]
I looked at how passkey/passwordless UX guidance (FIDO Alliance, Passkey Central) and browser permission dialogs handle this same problem, and the consistent pattern is:

1. Don't name the underlying concept (credentials) — describe the future action the user will take.
2. Anchor to a metaphor the user already has (a physical wallet with cards you choose between).
3. State it in concrete, forward-looking terms, not abstract capability language.

Applied to our case, something like:

"wants to be one of your wallet choices for future sign-ins"

or slightly more explicit:

 "wants to appear as an option when you're asked to sign in or share information on other websites"

Both drop "credential" entirely and describe a concrete use rather than an abstract permission grant. 

I'm partial to the first option, less wordy. What do you think?

**Comment by dlongley** [comment]
Yes, this language is an improvement. I do think we should try to make it even more concise -- and perhaps "sign in" should be removed and add "store":

"wants to appear as an option when you're asked to share or store information on other websites"

Though, "information" is pretty vague, I do find myself wanting to put "credentials" there. Anyway, we should see if we can cut down the suggestion I just made above to fewer words.

**Comment by dlongley** [comment]
This has been addressed. Further language tweaks can happen, but closing this.
